models/public/anti-spoof-mn3/anti-spoof-mn3.md

```diff
@@ -0,0 +1,78 @@
+# anti-spoofing-mn3
+
+## Use Case and High-Level Description
+
+`anti-spoofing-mn3` is an anti-spoofing binary classificator based on the [MobileNetv3](https://arxiv.org/abs/1905.02244), trained on the [Celeba_Spoof dataset](https://arxiv.org/abs/2007.12342). It's a small, light model, trained to predict whether or not a spoof RGB image given to the input. A lot of advanced techniques have been tried and selected the best suit options for the task.
+Original repository: https://github.com/kirillProkofiev/light-weight-face-anti-spoofing
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Classification                            |
+| GFlops                          | 2.93                                    |
+| MParams                         | 0.13                                    |
+| Source framework                | PyTorch\*                              |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| AUC    | 99.8%          | 99.8%         |
+| EER    | 2.25%          | 2.25%         |
+| APCER  | 0.68%          | 0.68%         |
+| BPCER  | 6.93%          | 6.93%         |
+| ACER   | 3.81%           | 3.81%          |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name: `input` , shape: [1x128x128x3], format: [BxCxHxW], where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: RGB.
+   Mean values: [0.5931,0.4690,0.4229], scale factor: [0.2471,0.2214,0.2157]
+
+### Converted Model
+
+Image, name: `input` , shape: [1x3x128x128], format: [BxCxHxW], where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: BGR.
+   Mean values: [151.2405,119.5950,107.8395], scale factor: [63.0105,56.4570,55.0035]
+
+## Output
+
+### Original Model
+
+Probabilities for two classes (0 class is a real person, 1 - is a spoof image). Shape: [1,2], format: [BxC],
+    where:
+
+    - B - batch size
+    - C - vector of probabilities.
+
+### Converted Model
```

Comment:
there is no difference between converted model and original, possibly it makes sense merge into one description here